modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -1,3 +1,90 @@
 = Identifiers
```

Comment:
I wonder if we can make this title more robust by adding what kind of identifiers, or identifiers for what? 
Helps those who scan titles....